de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -77,7 +77,8 @@
         @Override
         public void onFailure(Throwable caught) {
             if (caught instanceof HttpRedirectException) {
-                AgaveAuthPrompt prompt = AgaveAuthPrompt.getInstance();
+                final String uri = ((HttpRedirectException)caught).getLocation();
```

Comment:
Just for my own knowledge - is this redirect still related to Agave?  Is it different from what gets loaded in `userInfo.getAgaveRedirect()`?